jupyter-book/multi_omics/1_0_scATAC_intro.ipynb

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
<blockquote>We anticipate, that</blockquote><p> </p><p>No comma I think.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>